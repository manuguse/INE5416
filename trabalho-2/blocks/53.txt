1 7 7 7 7 15 15 20 20 20
1 2 8 8 7 15 12 21 21 20
1 2 8 8 12 12 12 22 20 20
2 2 3 9 13 13 12 12 25 20
3 3 3 3 14 14 17 17 17 23
4 4 3 10 10 10 10 17 17 23
4 5 3 10 10 11 18 18 23 23
5 5 5 10 11 11 18 23 23 23
6 5 5 11 11 16 19 24 24 24
6 6 6 6 11 11 19 24 24 24