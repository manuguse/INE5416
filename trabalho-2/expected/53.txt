3 1 4 2 5 2 3 1 3 6
2 4 3 4 3 1 6 2 1 5
1 3 1 2 1 2 5 1 7 4
2 1 7 1 2 1 4 3 1 2
4 6 5 2 1 2 3 2 5 6
3 2 3 6 4 3 7 1 4 5
1 6 1 5 2 4 2 3 7 4
4 5 3 1 5 2 1 2 3 1
3 1 2 6 3 1 2 4 5 6
2 4 1 5 1 7 1 2 1 3